platforms-serverless/serverless-framework-lambda/serverless.yml

```diff
@@ -19,7 +19,8 @@ functions:
 # only include the Prisma binary required on AWS Lambda while packaging
 package:
   patterns:
-    - '!node_modules/.prisma/client/query-engine-*'
-    - '!node_modules/.prisma/client/libquery_engine-*'
-    - 'node_modules/.prisma/client/libquery_engine-rhel-openssl-1.0.x.so.node'
-    - 'node_modules/.prisma/client/query-engine-rhel-openssl-1.0.x'
+    - '!node_modules/**'
```

Comment:
Uh completely excluding node_modules? Why?

Reply:
Related to https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1160130018
We generated in a  custom folder, so no need for node_modules

Reply:
https://github.com/prisma/ecosystem-tests/pull/3463#discussion_r1163418117